landsat/anaconda-project.yml

```diff
@@ -2,6 +2,7 @@
 name: landsat
 description: Datashading LandSat8 raster satellite imagery
 examples_config:
+  created: 2016-06-20
   maintainers:
   - jbednar
```

Comment:
There's no difference between authors and maintainers. IMHO that would be a little too detailed, and some work to find trace back the list of original authors and distinguish them with the maintainers... If that needs to be addressed, I'd suggest renaming it `contributors`, so that it sort of encompasses both?

Reply:
I assume these tags are parsed and displayed when we render the notebooks on the https://examples.pyviz.org/ page. Is `contributors` or `authors` parsed and displayed correctly? 

Looking here, I only see maintainers: https://github.com/holoviz-topics/examples/blob/main/dodo.py#L1019-L1023
It looks like I should also add a `last_updated` tag - it certainly applies.

Reply:
> contributors or authors parsed and displayed correctly

Nope, it's only `maintainers` and it is a required field. Nothing set in concrete though, but it needs some changes.

>  I should also add a `last_updated` tag

There are two options for `last_updated`:
- set it in the anaconda-project.yml file
- don't set it, it's computed automatically based on the last commit date that touched the project (anything in the project), see https://github.com/holoviz-topics/examples/blob/833126e41ac1db873209b9db1e97f0ed3408385c/doc/conf.py#L136

I think I prefer the automatic approach, I don't have much confidence in people updating that field 🙃  Myself first!

Reply:
I missed this - will remove it. Thanks!